.github/workflows/slurm_job_scripts/run_e2e_t5x_tests.sub

```diff
@@ -48,7 +48,7 @@ read -r -d '' setup_cmd <<EOF
 rm -rf ${E2E_TESTS_WORKSPACE_DIR}/* \
 && mkdir -p ${E2E_TESTS_WORKSPACE_DIR} \
 && mkdir -p ${TFDS_DATA_DIR} \
-&& JAXLIB=`wget -O - https://storage.googleapis.com/jax-releases/jaxlib_nightly_cuda_releases.html |grep cuda11|grep -oP 'ht.*?whl'|sort|tail -n 1` \
+&& JAXLIB=`wget -O - https://storage.googleapis.com/jax-releases/jaxlib_nightly_cuda_releases.html 2>/dev/null |grep cuda11|grep -oP 'ht.*?whl'|sort|tail -n 1` \
```

Comment:
FYI: This undo something that shouldn't be undone.
I see other places changes like this.
I just wanted to warn you that this can't be merged as is.

Reply:
Sync to head and this change disappear.
Thanks a lot for the catch.